tasks/post_copy_request_to_queue/post_copy_request_to_queue.py

```diff
@@ -171,7 +166,6 @@ def exponential_delay(base_delay: int, exponential_backoff: int = 2) -> int:
 def query_db(key_path: str, bucket_name: str) -> List[Dict[str, str]]:
     """
     Connect and query the recover_file status table return needed metadata for posting to the recovery status SQS Queue.
-
```

Comment:
Not sure why you are removing these empty lines. They provide whitespace making the description easier to read.

Reply:
added back